devices/stm32f7x6.yaml

```diff
@@ -77,6 +77,15 @@ CRYP:
     _merge:
       - "IV*"
 
+SYSCFG:
+  MEMRMP:
+    _delete:
+      - FB_MODE
+  PMC:
+    _delete:
+      - PN*
+      - I2C*
```

Comment:
Looking at the [memory map](https://github.com/stm32-rs/stm32-rs-mmaps/compare/pr-612-1212829#diff-f146746c79235e17e7a8674bbb31537af0c2d61e3d3552ab34f078fed0fd67d8) it doesn't seem like these deletions are doing anything?